project.yml

```diff
@@ -1,5 +1,13 @@
+parameters: {}
+environment: {}
 packages:
   - name: sample
+    environment: {}
+    parameters: {}
     actions:
       - name: hello
+        main: ''
         runtime: 'go:default'
+        environment: {}
+        parameters: {}
```

Comment:
Are all the additions here necessary? Seems like added noise.

Reply:
I like the env placeholder. I think they’re are generated by the cli on a new project init.

Reply:
I added them to the sample, so that newcomers could easily see that there are options to add variables here. Otherwise its not really clear unless you find our pdocs. Given that it's a sample app it made sense to me but i dont feel strongly either way.

Reply:
I don't feel strongly either. Fine to leave as-is to me.